utils/_context/_scenarios/__init__.py

```diff
@@ -693,6 +693,14 @@ def all_endtoend_scenarios(test_object):
 
     external_processing = ExternalProcessingScenario("EXTERNAL_PROCESSING")
 
+    # It's VERY important to match the exact name of the property and the name
+    my_nice_scenario = EndToEndScenario(
+        name="MY_NICE_SCENARIO",
+        weblog_env={"DD_VERY_NICE_FEATURE": "wow wow wee wow",},
+        scenario_groups=[ScenarioGroup.VERY_COOL_SCENARIO_GROUP],
```

Comment:
This feature is optional, and probably too secondary regarding other point you'll present, I suggest not having that.

Reply:
Removing scenario groups, you mean?